chrome/content/zotero/xpcom/quickCopy.js

```diff
@@ -91,7 +100,7 @@ Zotero.QuickCopy = new function() {
 			var urlPath = nsIURI.path;
 		}
 		catch (e) {
-			return Zotero.Prefs.get("export.quickCopy.setting");
+			return quickCopyPref;
 		}
```

Comment:
I don't know that `hostPort` or `path` can throw. `newURI` throws if the URI is invalid. That's probably what's supposed to be in the `try`.